R/mainTests.r

```diff
@@ -1042,12 +1100,44 @@ not_table_tool <- function(data) {
       "result" = "ANCILLARY"
     )
   } else {
-    output <- list(
-      "message" = "This file contains rows that will be used in the table tool for EES.",
-      "result" = "PASS"
-    )
-  }
+    potential_ignored_rows <- data %>%
+      filter(geographic_level %in% geography_matrix[13:16, 1]) %>%
+      nrow()
+
+    if (potential_ignored_rows == 0) {
+      output <- list(
+        "message" = "No rows in the file will be ignored by the EES table tool.",
+        "result" = "PASS"
+      )
+    } else {
+      levels_present <- data %>%
+        distinct(geographic_level)
 
+      if (nrow(levels_present) == 1 && data$geographic_level[1] %in% geography_matrix[13:14, 1]) {
+        output <- list(
+          "message" = "No rows in the file will be ignored by the EES table tool.",
+          "result" = "PASS"
+        )
+      } else {
+        if (geography_matrix[13, 1] %in% levels_present$geographic_level && geography_matrix[14, 1] %in% levels_present$geographic_level) {
+          output <- list(
+            "message" = paste(geography_matrix[13, 1], "and", geography_matrix[13, 1], "data has been mixed - please contact the Statistics Development Team."),
```

Comment:
One needs pointing to "Provider" at geography_matrix[14,1] as currently repeats "School"
![image](https://user-images.githubusercontent.com/75032474/127831703-8414c9e0-467a-43fc-b1cf-51984573e520.png)

Reply:
Same as Laura's comment - think I've just pushed a fix